### PR TITLE
First release identifier is v0.1.0

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
This PR is mostly a global search-and-replace of "step 1" to "v0.1.0".  I also took the liberty of adding a parameter table to the instantiation sub-section.  This will need to be expanded, so it is just a placeholder for now.

### CODE REVIEW

File: docs/design_spec/source/cva6_system.rst [hunk 1]
```diff
@@ -197,22 +216,22 @@ CVA6 memory interface complies with AXI5 specification including the Atomic_Tran
 
 CVA6 coprocessor interface complies with CV-X-IF protocol specification as defined in [CV-X-IF].
 
-The CV32A4-step1 core is fully synthesizable. It has been designed mainly for ASIC designs, but FPGA synthesis is supported as well.
+The CV32A6 cwv0.1.0 core is fully synthesizable. It has been designed mainly for ASIC designs, but FPGA synthesis is supported as well.
```
Comment (JeanRochCoulon) [inline docs/design_spec/source/cva6_system.rst:219]: cwv0.1.0 => v0.1.0

**Comment by JeanRochCoulon** [comment]
@MikeOpenHWGroup, just a small syntax error in the text. That's all right for me !

**Review by JeanRochCoulon — CHANGES_REQUESTED** [review]
syntax erorr to be fixed, otherwise, it is ok for me !

File: docs/design_spec/source/cva6_system.rst (hunk 1 above)
Comment (MikeOpenHWGroup) [reply docs/design_spec/source/cva6_system.rst]: Oops!  :blush: Good catch.  Fixed.

**Review by JeanRochCoulon — APPROVED** [review]

**Pull request merged**